src/DataCite.jl

```diff
@@ -24,11 +24,11 @@ function paper_cite(::DataCite, mainpage)
     paper_cite
 end
 
-function get_urls(repo::DataCite, page)
-    urls = ["PUT DOWNLOAD URL HERE"]
-    info("DataCite based generation can only generate partial registration blocks, as DataCite metadata does not (currently) include the URL to the resource. You will have to edit in the URL after generation.")
-    urls
-end
+# function get_urls(repo::DataCite, page)
+#     urls = ["PUT DOWNLOAD URL HERE"]
+#     info("DataCite based generation can only generate partial registration blocks, as DataCite metadata does not (currently) include the URL to the resource. You will have to edit in the URL after generation.")
+#     urls
+# end
```

Comment:
Because, `"PUT DOWNLOAD URL HERE"` is not a valid url and when comparing the length, a valid url array with length 1 will get ignored if this goes in. This can probably shift to the final makeup of the registration block, in which `missing` data_urls will be replaced with the `"PUT DOWNLOAD URL HERE"`.

Reply:
Cool makes sense.
I think even just leaving the URL as missing is fine.
Delete this rather than just commenting it out.